amplitude/build.gradle

```diff
@@ -0,0 +1,67 @@
+
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+        google()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.2.0'
+        classpath 'com.f2prateek.javafmt:javafmt:0.1.6'
+        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.11.0'
+    }
+}
+
+// Use -Prelease or $ORG_GRADLE_PROJECT_RELEASE="true" to mark the project as a release
+ext.isRelease = hasProperty('release')
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.f2prateek.javafmt'
+
+android {
+    compileSdkVersion 28
+    buildToolsVersion '28.0.2'
+
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 28
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+    }
+}
+
+dependencies {
+    repositories {
+        mavenCentral()
+        jcenter()
+        google()
+    }
+
+    api 'com.segment.analytics.android:analytics:4.5.0-beta.0'
+    api 'com.amplitude:android-sdk:2.23.2'
+
+    testImplementation 'com.segment.analytics.android:analytics-tests:4.3.1'
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'com.android.support.test:runner:1.0.2'
+    testImplementation 'org.mockito:mockito-core:2.28.0'
+
+    // Required for local (non-android) testing
+    testImplementation 'org.json:json:20180813'
+}
+
+//apply from: rootProject.file('gradle/attach-jar.gradle')
```

Comment:
We'll need to figure out how we want to do all these plugins in the monorepo. In an ideal world, the destinations can be published independently.

Reply:
Okay, I found where all these plugins are located and I will get it working